Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinationTestData.kt

```diff
@@ -173,7 +175,8 @@ class VaccinationTestData @Inject constructor(
 
     val personBVac1CertificateData = DccData(
         certificate = personBVac1Certificate,
-        header = personBVac1CertificateHeader
+        header = personBVac1CertificateHeader,
+        certificateJson = "{\"v\":[{\"ci\":\"01DE/00001/1119305005/3H24U2KVOTPCSINK7N64F2OB9#S\",\"co\":\"DE\",\"dn\":1,\"dt\":\"2021-04-20\",\"is\":\"Bundesministerium für Gesundheit - Test01\",\"ma\":\"ORG-100001417\",\"mp\":\"EU/1/20/1525\",\"sd\":1,\"tg\":\"840539006\",\"vp\":\"1119305005\"}],\"dob\":\"1966-11-11\",\"nam\":{\"fn\":\"Johnson Gültig\",\"gn\":\"Boris\",\"fnt\":\"JOHNSON<GUELTIG\",\"gnt\":\"BORIS\"},\"ver\":\"1.0.0\"}",
```

Comment:
With """  you can avoid escaping everything

Reply:
Good point, forgot about that yesterday.
I just copy pasted it into the `""` and Android Studio did all the escaping for me :grinning:, I'd leave it like this for now, if that is okay.